projects/demo-playwright/tests/kit/input-date/input-date.spec.ts

```diff
@@ -0,0 +1,146 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+import {TUI_PLAYWRIGHT_MOBILE_USER_AGENT} from '../../../playwright.options';
+
+test.describe('InputDate', () => {
+    test.use({viewport: {width: 750, height: 500}});
+
+    test.describe('Examples', () => {
+        test('correct filler display for size', async ({page}) => {
+            await tuiGoto(page, '/components/input-date');
+
+            const api = new TuiDocumentationPagePO(page);
+            const example = api.getExample('#sizes');
+
+            await example.scrollIntoViewIfNeeded();
+            await api.prepareBeforeScreenshot(':not(#sizes)');
+
+            for (const size of ['s', 'm', 'l']) {
+                const input = example
+                    .locator(`[tuiTextFieldSize="${size}"]`)
+                    .locator('[automation-id="tui-primitive-textfield__native-input"]');
+
+                await input.click();
+                await expect(page).toHaveScreenshot(`01-01-input-date-${size}.png`);
+                await input.clear();
+                await expect(page).toHaveScreenshot(`01-02-input-date-${size}.png`);
+                await input.focus();
+                await page.keyboard.type('01.');
+                await expect(page).toHaveScreenshot(`01-03-input-date-${size}.png`);
+                await input.focus();
+                await page.keyboard.type('06.1994');
+                await expect(page).toHaveScreenshot(`01-04-input-date-${size}.png`);
```

Comment:
![01-02-input-date-s-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/20aae922-72e0-4bb8-abfc-e5dfaf643f10)

Reply:
You can set different viewport size for different `test.describe()`

Reply:
fixed